custom_components/tahoma/cover.py

```diff
@@ -28,13 +28,13 @@ async def async_setup_entry(
     entities = [
         Awning(device.deviceurl, coordinator)
         for device in data["platforms"].get(COVER)
-        if device.ui_class == "Awning"
+        if device.ui_class == "Awning" and device.widget != "AwningValance"
     ]
 
     entities += [
         VerticalCover(device.deviceurl, coordinator)
         for device in data["platforms"].get(COVER)
-        if device.ui_class != "Awning"
+        if device.ui_class != "Awning" or device.widget == "AwningValance"
```

Comment:
I think we will have a reverse position issue.

Reply:
It has been confirmed by 2 users that it now works like a charm. They had the reverse position issue with the `Awning`.